### PR TITLE
js, html and css files for Change password mockup

### PR DESCRIPTION
**Pull request by sharkoo0**
The file "myuser.html" has to be deleted when the actual "My User" file is available.

### CODE REVIEW

**Comment by sharkoo0** [comment]
The CSS has to be fixed too.

File: changePassword.html [hunk 1]
```diff
@@ -1,7 +1,32 @@
 <html>
```
Comment (yavorp) [inline changePassword.html:2]: Missing <!DOCTYPE html> for refference you can check https://www.w3schools.com/tags/tag_doctype.asp

File: css/changePassword.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.content { 
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 50px;
+    margin-bottom: auto;
+    width: 200px;
+    /* background-color: aquamarine; */
```
Comment (yavorp) [inline css/changePassword.css]: Do not leave commented code in the repo

File: changePassword.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+function next() {
+    let firstField = document.getElementsByClassName('pass');
+    let secondField = document.getElementsByClassName('conf-pass');
```
Comment (yavorp) [inline changePassword.js]: It is useful to use `let` when it is not possible to use `const`, so here variables should be declared with `const`

**Review by yavorp — CHANGES_REQUESTED** [review]

File: changePassword.js (hunk 3 above)
Comment (sharkoo0) [reply changePassword.js]: But in this case I want to change one of the properties of these elements. That's why I used **let**.

Comment (yavorp) [reply changePassword.js]: HTML element is a **structural** type. So you can declare button as a constant and change the inner properties. `const` protects you from unintentional change of the reference of the object.

File: changePassword.html [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
-    <body>
-        <h1>zadara</h1>
-        <h1>kopr</h1>
-        <h1>dobre tii</h1>
-    </body>
+
+<head>
+    <title>Change password</title>
+    <link rel="stylesheet" type="text/css" href="css/changePassword.css">
+    <script src="./changePassword.js"></script>
```
Comment (ngadzheva) [inline changePassword.html:7]: Добре е елементът `script` да има атрибут `defer`, за да се избегнат неочаквани грешки.

File: changePassword.html [hunk 5]
```diff
@@ -1,7 +1,34 @@
...
+
+</head>
+
+<body>
+    <nav class="primary">
+        <ul>
+            <li #1>
+                <a href="#" class="prev" onclick="back()">&laquo; Back</a>
```
Comment (ngadzheva) [inline changePassword.html:15]: Избягвайте използването на `onclick` атрибути на HTML елементите. По-добре си закачайте `eventListener`-и за елементите в JS.

**Pull request merged**